index.html

```diff
@@ -0,0 +1,178 @@
+				</div>
+				<div class="reference">Mirror Lake, Yosemite Valley, United States by Quentin Dr</div>
+			</div>
+		</div>
+		<div class="thumbnails-popUp">
+			<div class="thumbnails-imgs actual" onclick="showSlide(idx,idx=1)"><img src="https://images.unsplash.com/photo-1523977843538-0650e8cb2e7d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=a43d0d3984b7a6e57716c15ee6fe5797&auto=format&fit=crop&w=1050&q=80"></div>
+			<div class="thumbnails-imgs actual" onclick="showSlide(idx,idx=2)"><img src="https://images.unsplash.com/photo-1491904768633-2b7e3e7fede5?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=b201ea4f1683a8e092ff153c1b423e78&auto=format&fit=crop&w=1189&q=80"></div>
+			<div class="thumbnails-imgs actual" onclick="showSlide(idx,idx=3)"><img src="https://images.unsplash.com/photo-1516524609276-3cb95aa22190?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=3d79a0b6dc9683effb116d7e9763bfa2&auto=format&fit=crop&w=1225&q=80"></div>
+			<div class="thumbnails-imgs actual" onclick="showSlide(idx,idx=4)"><img src="https://images.unsplash.com/photo-1517483141275-296bb15890b2?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=f6fb898eac03ab9abd1137fa691ebe82&auto=format&fit=crop&w=1051&q=80"></div>
+			<div class="thumbnails-imgs actual" onclick="showSlide(idx,idx=5)"><img src="https://images.unsplash.com/photo-1480499484268-a85a2414da81?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=6d765c5039bd6d36022739e3a6552c44&auto=format&fit=crop&w=1055&q=80"></div>
+		</div>
+		<div class="Option-bar">
+			<div class="btn info" onclick="showInfo()"></div>
+			<div class="thumbnails btn" onclick="showThumbnails()"></div>
+			<div class="counter">0/0</div>
+			<div class="btn prev" onclick="prevSlide()"></div>
+			<div id="playBtn" class="btn play" onclick="play=!play; playSlides()"></div>
+			<div class="btn next" onclick="nextSlide()"></div>
+		</div>
+	</div>
+<script>
+
+	//Elements
+	const imgs = document.querySelectorAll('.imgs');
+	const container = document.querySelector('.slideShow-container');
+	const thumbnailImgs = document.querySelectorAll('.thumbnails-imgs');
+	const counter = document.querySelector('.counter');
+	const popUp = document.querySelector('.thumbnails-popUp');
+	const infoBoxes = document.querySelectorAll('.paragraph');
+	const btnPlay = document.querySelector('#playBtn');
+	const refereces = document.querySelectorAll('.reference');
+	const titlesH3 = document.querySelectorAll('h3');
+	const SlideTimeout = 2000;
+
+	var intervalSlides;
+	var idx = 1;
+	var up = false;
+	var infoUp = false;
+	var refUp = false;
+	var play = false;
+	var firstTime = true;
+
+	playSlides();
+
+	//Function
+	function nextSlide() {
+		showSlide(idx,idx += 1);
+	}
+
+	function prevSlide() {
+		showSlide(idx,idx -= 1);
+	}
+
+	//Check Funtions
+	function checkIdx(n) {
```

Comment:
this name doesn't describe what the function is doing. We could discuss this

Reply:
*I would like to discuss this one*